src/IterativeSolvers.fypp

```diff
@@ -41,7 +50,7 @@ module lightkrylov_IterativeSolvers
         #:endfor
     end interface
 
-   interface eigs
+    interface eigs
```

Comment:
Extra whitespace ?

Reply:
I think I have a different tab setting than you, at least for some of the files. So sometimes extra whitespaces may appear. 
But in this particular case, I think there was simply a mismatch between the indentations. OCD alarm.